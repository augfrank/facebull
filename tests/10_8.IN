M0   C0   C1  2375
M1   C0   C2  2016
M2   C0   C3  2934
M3   C0   C4  2572
M4   C0   C5  2173
M5   C0   C6  2193
M6   C0   C7  2738
M7   C0   C8  2099
M8   C0   C9  2449
M9   C1   C0  2917
M10   C1   C2  2796
M11   C1   C3  2826
M12   C1   C4  2238
M13   C1   C5  2090
M14   C1   C6  2945
M15   C1   C7  2736
M16   C1   C8  2669
M17   C1   C9  2594
M18   C2   C0  2871
M19   C2   C1  2766
M20   C2   C3  2136
M21   C2   C5  2018
M22   C2   C6  2016
M23   C2   C8  2625
M24   C2   C9  2299
M25   C3   C0  2154
M26   C3   C1  2795
M27   C3   C4  2936
M28   C3   C5  2664
M29   C3   C6  2268
M30   C3   C7  2016
M31   C3   C8  2663
M32   C3   C9  2075
M33   C4   C0  2096
M34   C4   C1  2919
M35   C4   C2  2795
M36   C4   C5  2407
M37   C4   C6  2885
M38   C4   C7  2833
M39   C4   C8  2754
M40   C4   C9  2958
M41   C5   C0  2377
M42   C5   C1  2397
M43   C5   C2  2739
M44   C5   C3  2946
M45   C5   C4  2909
M46   C5   C6  2649
M47   C5   C7  2433
M48   C5   C8  2125
M49   C5   C9  2939
M50   C6   C0  2269
M51   C6   C1  2095
M52   C6   C3  2382
M53   C6   C4  2083
M54   C6   C5  2781
M55   C6   C7  2648
M56   C6   C8  2408
M57   C6   C9  2597
M58   C7   C1  2428
M59   C7   C3  2348
M60   C7   C5  2290
M61   C7   C6  2004
M62   C7   C8  2119
M63   C7   C9  2056
M64   C8   C0  2745
M65   C8   C1  2721
M66   C8   C2  2687
M67   C8   C3  2605
M68   C8   C4  2217
M69   C8   C5  2208
M70   C8   C6  2374
M71   C8   C7  2522
M72   C8   C9  2226
M73   C9   C0  2148
M74   C9   C2  2509
M75   C9   C3  2811
M76   C9   C4  2450
M77   C9   C5  2345
M78   C9   C6  2250
M79   C9   C7  2745
M80   C9   C8  2448
