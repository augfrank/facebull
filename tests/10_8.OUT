21097
1 13 20 30 33 48 51 63 68 78
